rpm_base_4repos_rerun_copy/generate.sh

```diff
@@ -0,0 +1,41 @@
+sudo systemctl stop httpd pulp_workers pulp_resource_manager pulp_celerybeat
+mongo pulp_database --eval "db.dropDatabase()"
+sudo rm -rf /var/lib/pulp/content
+sudo rm -rf /var/lib/pulp/published
+sudo -u apache pulp-manage-db;
+sudo systemctl start httpd pulp_workers pulp_resource_manager pulp_celerybeat
+pulp-admin login -u admin -p admin
+pulp-admin rpm repo create --repo-id rpm-empty
+pulp-admin rpm repo create --repo-id rpm-empty-for-copy
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/rpm-with-modules/ --repo-id rpm-with-modules --download-policy on_demand
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/rpm-distribution-tree/ --repo-id rpm-distribution-tree --download-policy on_demand
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/srpm-unsigned/ --repo-id srpm-unsigned --download-policy on_demand 
+pulp-admin rpm repo sync run --repo-id rpm-with-modules
+pulp-admin rpm repo sync run --repo-id rpm-distribution-tree
+pulp-admin rpm repo sync run --repo-id srpm-unsigned
+
+pulp-admin rpm repo remove rpm --repo-id rpm-with-modules --str-eq name=bear
+pulp-admin orphan remove --all
```

Comment:
why are we running orphan clean up here?

Reply:
so the removed package is not in pulp 2 or on the filesystem.

FWIW, this is already this way in the rpm_base_4repos_rerun_changes snapshot. This part is unchanged.
I marked what changed in this one. see my comment with "here".